---
title: ARTIC/WACCBIP training - Accra, Ghana, 26th to 31st March 2026
keywords: workshop training
sidebar: artic_sidebar
toc: false
permalink: /workshops/accra_march_2026
folder: workshops
status: open
tags: [workshop]
---

## ***Overview***

The ARTIC methodology, which uses a tiled amplicon-based sequencing approach, has been instrumental in responding to outbreaks and epidemics like those of Ebola, Zika, and SARS-CoV-2. The technology is fast, portable, and robust, enabling real-time surveillance and epidemiological tracking of virus outbreaks.

The training is a collaboration between WACCBIP and the ARTIC network and will offer hands-on experience spanning the full workflow, from sample preparation to data interpretation, simulating viral genome evolution using synthetic nucleic acid controls. These materials are biologically inert and safe for handling, and provide scenario-realistic training. The participants will work on sequencing genomes based on a simulated outbreak scenario.

## ***Who is this course for?***

This course is intended for researchers and staff in academic and/or public health microbiology laboratories performing infectious diseases research, real-time virus surveillance, and/or outbreak epidemiology.   
Participants should ideally:

* Have a foundational knowledge in molecular biology
* Be interested or have prior experience in applying genomics during outbreaks or public health investigations

### ***Learning outcomes***

On completion of the course, participants can expect to be able to:

* Explain the advantages (and limitations) of portable viral sequencing technology for real-time epidemiological surveillance
* Understand sample preparation and the use of appropriate experimental controls within the context of viral sequencing
* Understand the principles of primer design for viral genome sequencing and perform PCR amplification of viral genomes
* Prepare and load sequencing libraries for ONT sequencing
* Monitor the sequencing run output (with MinKNOW and RAMPART)
* Process raw sequencing data through bioinformatics pipelines, interpret the quality control outputs, and generate reports
* Understand the components of a phylogenetic tree and interpret phylogenetic trees and epidemiological data

### ***Tentative agenda***
## ARTIC Workshop Schedule

| Session | Time | Day 1 - Thursday | Day 2 - Friday | Day 3 - Saturday | | Day 4 - Monday | Day 5 - Tuesday |
|---------|------|------------------|----------------|------------------|---|----------------|-----------------|
| | 08:30 | Arrival | Arrival | Arrival | | Arrival | Arrival |
| **AM 1** | 09:00 | Introductions - ARTIC, Instructors & participants | Overview of next steps in lab protocol | Demo/showcase of ONT flow cells, MinION, MinKNOW, EPI2ME | | Intro to phylogenetics and phylodynamics | Recap of the week |
| | 09:30 | Introduction to course, course overview | Theory of SPRI, EP, barcoding | *(continued)* | | *(continued)* | Computational analysis wrap up and Discussion |
| | 10:00 | Real world data talk | *(continued)* | *(continued)* | | *(continued)* | *(continued)* |
| **Break** | 10:40 | | | | | | |
| **AM 2** | 11:00 | Introduction to genomic epidemiology | Intro to lab in a suitcase | Lab session 4 (adapter ligation, cleanup, library prep, flow cell checks, priming, loading) | | Overview of ARTIC bioinformatics (fieldbioinformatics, Amplicon-nf, other tools) | Discussion session 1: Result interpretation, deriving actionable epidemiological insights, conducting phylogenetic analyses |
| | 11:30 | *(continued)* | Lab session 2 (PCR pooling, cleanup, quantification) | *(continued)* | | *(continued)* | *(continued)* |
| | 12:00 | Intro to nanopore sequencing and ARTIC protocol overview | *(continued)* | *(continued)* | | Computational session 1 (Running amplicon-nf) | *(continued)* |
| | 12:30 | *(continued)* | *(continued)* | *(continued)* | | *(continued)* | *(continued)* |
| **Lunch** | 13:00 | | | | | | |
| | 13:30 | | | | | | |
| **PM 1** | 14:00 | Principles of safe sample handling, H&S, extraction of viral nucleic acids, RT-PCR | Lab session 3 (EP, Barcoding, cleanup) | Lab session 4 cont'd | | Computational session 2 (going through results, SNP calling, building a tree, other analyses) | Discussion session 2: BEAST/Phylodynamics Demo |
| | 14:30 | *(continued)* | *(continued)* | *(continued)* | | *(continued)* | *(continued)* |
| | 15:00 | Outbreak scenario overview | *(continued)* | *(continued)* | | *(continued)* | *(continued)* |
| **Break** | 15:40 | | | | | | |
| **PM 2** | 16:00 | Overview of week/Group allocation | Lab session 3 cont'd | Sequencing monitoring, RAMPART, diagnosing problems, laptop bootcamp | | Computational session 2 cont'd | Discussion session 3: use of genomic data for real time epidemiology |
| | 16:30 | Lab session 1 (samples, samplesheet, multiplex PCR) | *(continued)* | *(continued)* | | *(continued)* | *(continued)* |
| | 17:00 | *(continued)* | *(continued)* | *(continued)* | | *(continued)* | Wrap up, Certificates, and awards |
| | 17:30 | *(continued)* | *(continued)* | *(continued)* | | *(continued)* | *(continued)* |
| **End of day** | 18:00 | | | | | | Feedback collection |
| | 18:30 | | | | | | |

**Note:** Sessions marked with *(continued)* indicate that the activity spans multiple time slots. Day 3 (Saturday) and Day 4 (Monday) are separated by a day off (Sunday).

### ***Requirements***

The participants will require a laptop computer for individual or shared use.

***Application***  
Applications should be submitted at [https://redcap.link/bzxttagu](https://redcap.link/bzxttagu) by **10th January 2026**. All enquiries regarding this workshop should first be directed to Lady Baaba Donkoh, [lbdonkoh@ug.edu.gh](mailto:lbdonkoh@ug.edu.gh). For participants who require a visa to travel to Ghana, they can be issued an invitation letter by Lady Baaba Donkoh, [lbdonkoh@ug.edu.gh](mailto:lbdonkoh@ug.edu.gh), which can be used as a supporting document for your application. More information on immigration can be accessed via [https://gis.gov.gh/](https://gis.gov.gh/).


[image1]: 